src/otx/cli/train.py

```diff
@@ -1,31 +1,53 @@
 # Copyright (C) 2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
-#
+
 """CLI entrypoint for training."""
 # ruff: noqa
```

Comment:
Do we need this?

Reply:
Yes I was trying to get rid of this, but unless the type of the callbacks and the logger in TrainConfig is finalized, I'm ignoring it for now due to the mypy issue.
I expect this will be resolved once the type issues with callbacks and loggers are resolved.